shared/test/test_redis_table.rb

```diff
@@ -153,8 +161,8 @@ def test_delete_many
 
   def test_expiration
     # Test with a 1-second expire time
-    expire_time = 2
-    margin_time = 0.2
+    expire_time = 0.01
+    margin_time = 0.002
```

Comment:
i assume this change is just to make the tests run more quickly since we're doing actual sleep calls now?